lsbi/stats.py

```diff
@@ -43,6 +45,33 @@ def _bar(self, indices):
         k[indices] = False
         return k
 
+    def bijector(self, x, inverse=False):
+        """Bijector between U([0, 1])^d and the distribution.
+
+        - x in [0, 1]^d is the hypercube space.
+        - theta in R^d is the physical space.
+
+        Computes the transformation from x to theta or theta to x depending on
+        the value of inverse.
+
+        Parameters
+        ----------
+        x : array_like, shape (..., d)
```

Comment:
Does this want/benefit stricter shape checking? I note for example if I have a 100D data likelihood I can

`model.likelihood(theta).bijector(np.random.rand(1)[...,None])`

i.e. I'm passing something of shape (1,1), rather than (100,) or (1,100) and it returns a valid data draw, but I'm not sure what this actually is!